scripts/biomedical/VirtualMetabolicHuman/format_reaction.py

```diff
@@ -0,0 +1,221 @@
+import pandas as pd
```

Comment:
for all these py files, consider running pylint and yapf: https://github.com/datacommonsorg/data#code-quality

Reply:
Done !